app/economy/models.py

```diff
@@ -66,12 +66,12 @@ def default(self, obj):
 
 def get_time():
     """Get the local time."""
-    return localtime(timezone.now())
+    return timezone.now()
 
 
 def get_0_time():
     """Get the local time."""
-    return localtime(timezone.datetime(1970, 1, 1).replace(tzinfo=pytz.utc))
+    return timezone.datetime(1970, 1, 1).replace(tzinfo=pytz.utc)
 
 
 class SuperModel(models.Model):
```

Comment:
^ can these be undone ?

Reply:
This change was by @zoek1, so I'll defer to him about if it should be undone or kept

Reply:
done